### PR TITLE
Integer domains

### PR DESCRIPTION
**Pull request by N-J-Martin**

### CODE REVIEW

File: docs/bits/type/int.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+defined by
+```int (<ranges list>)```
+
+If no range is provided, then represents infinite domain of integers.
+
+An integer range is either a single integer ```int(n)```, or a list of sequential integers with a given lower and upper bound 
+ ```int(l..u)```
+
+Values must be between -2<sup>62</sup> + 1 and 2<sup>62</sup> - 1
```
Comment (ozgurakgun) [inline docs/bits/type/int.md:11]: Maybe also say that ranges can be combined using a comma. so something like `int(n,l..u,k..r)` is valid.

File: docs/bits/keyword/letting_domain.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ letting x be domain int(1..10)
 Types of domains:
 
 - [boolean](https://github.com/conjure-cp/conjure/blob/main/docs/bits/type/L_bool.md)
-- [integer](...)
+- [integer](https://github.com/conjure-cp/conjure/blob/main/docs/bits/type/int.md)
```
Comment (ozgurakgun) [inline docs/bits/keyword/letting_domain.md:14]: Why not L_int?

**Comment by N-J-Martin** [comment]
hi @hz66-404 ! This should be ready to be reviewed now.

**Comment by hz66-404** [comment]
the new commits look good to me and the link looks fine

File: docs/bits/keyword/letting_domain.md (hunk 2 above)
Comment (ozgurakgun) [reply docs/bits/keyword/letting_domain.md:14]: this link still says int.md

Comment (N-J-Martin) [reply docs/bits/keyword/letting_domain.md]: should be fixed now

**Comment by ozgurakgun** [comment]
I see you used v0.0.10 in this notebook :)

**Pull request merged**